.github/dependabot.yml

```
@@ -10,3 +10,8 @@ updates:
   schedule:
     interval: daily
   open-pull-requests-limit: 10
+- package-ecosystem: npm
+  directory: "/web"
+  schedule:
+    interval: daily
```

```suggestion
    interval: weekly
```

Will check for updates on Mondays at 5am UTC